frontend/packages/kubevirt-plugin/integration-tests/tests/cdi-upload.scenario.ts

```diff
@@ -0,0 +1,54 @@
+import * as path from 'path';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { click } from '@console/shared/src/test-utils/utils';
+import { clickNavLink } from '@console/internal-integration-tests/views/sidenav.view';
+import {
+  createItemButton,
+  isLoaded,
+  saveChangesBtn,
+} from '@console/internal-integration-tests/views/crud.view';
+import { inputPVCName, inputPVCSize } from '@console/internal-integration-tests/views/storage.view';
+import { storageclassDropdown } from '@console/ceph-storage-plugin/integration-tests/views/pvc.view';
+import { CDI_UPLOAD_TIMEOUT_SECS } from './utils/constants/common';
+import * as cdiUploadView from '../views/cdiUploadView';
+
+describe('KubeVirt CDI Upload', () => {
+  const pvc = { name: 'upload-pvc', size: 100 };
+
+  it(
+    'ID(CNV-4718) Upload data to CDI',
+    async () => {
+      const fileToUpload = '../views/cdiUploadView.ts';
+      const absolutePath = path.resolve(__dirname, fileToUpload);
+
+      await clickNavLink(['Storage', 'Persistent Volume Claims']);
+      await isLoaded();
+      await click(createItemButton);
+      await click(cdiUploadView.uploadCdiFormButton);
+
+      await browser.wait(
+        until.textToBePresentInElement(
+          $('.co-m-pane__heading'),
+          'Upload Data to Persistent Volume Claim',
+        ),
+      );
+
+      await inputPVCName.sendKeys(pvc.name);
+      await inputPVCSize.sendKeys(pvc.size);
+      await click(cdiUploadView.unitDropdown);
```

Comment:
can we use selectDropdownOption here instead?

Reply:
`selectDropdownOption` needs a specific option.
 since the classes is dynamic, its safer to just pick the first option.

Reply:
It seems the same function is used by selectNetworkTypeByID which is testing the same underlying component Dropdown. 
Is there some issue with using the same approach here?